drawers/cMouseDrawer.cpp

```diff
@@ -190,58 +190,58 @@ void cMouseDrawer::drawToolTipGeneralInformation(cAbstractStructure * theStructu
 	} else {
 		sprintf(description, "%s", structureType.name);
 	}
-	textWriter->write(description, makecol(255, 255, 0));
-	textWriter->writeWithTwoIntegers("Hitpoints : %d/%d", theStructure->getHitPoints(), theStructure->getMaxHP());
-	textWriter->writeWithOneInteger("Armor : %d", theStructure->getArmor());
-	textWriter->writeWithOneInteger("Protected : %d%%", (100-theStructure->getPercentageNotPaved()));
+	textWriter.write(description, makecol(255, 255, 0));
+	textWriter.writeWithTwoIntegers("Hitpoints : %d/%d", theStructure->getHitPoints(), theStructure->getMaxHP());
```

Comment:
we can - eventually - get rid of `writeWith...` by combining `fmt::format` with `write()`. Not for now though :)